charts/argo-workflows/values.yaml

```diff
@@ -502,7 +502,8 @@ artifactRepository:
     secretKeySecret:
       # name: <releaseName>-minio
       key: secretkey
-    insecure: true
+    # insecure will disable TLS. Primarily used for minio installs not configured with TLS
+    insecure: false
```

Comment:
According to [upstream's doc](https://github.com/argoproj/argo-workflows/blob/master/docs/workflow-controller-configmap.yaml#L94-L95), this flag is used for specific case, so I added a note. 🙋